packages/salesforcedx-vscode-apex/package.json

```diff
@@ -153,7 +153,7 @@
         },
         {
           "command": "sfdx.force.test.view.runSingleTest",
-          "when": "view == sfdx.force.test.view && viewItem =~ /(apexTest)(_.*|\\b)/",
+          "when": "view == sfdx.force.test.view && viewItem =~ /(apexTest)(_.*||\\b)/",
```

Comment:
dumb reg ex q.  Seems like they moved to js style regular expressions.  So did it just change for a|b  -> a or b  to a||b -> a or b?

Reply:
it seems so! According to [this issue](https://github.com/microsoft/vscode/issues/175540) and [this page](https://code.visualstudio.com/api/references/when-clause-contexts)

Reply:
hum...interestingly the matches example on that page uses the single `|` 
![Screenshot 2023-03-24 at 12 01 35 PM](https://user-images.githubusercontent.com/76090802/227592430-8ef6e465-1061-495e-922e-97566d73c182.png)

Reply:
Oh right... mm then the || would be for other cases?
<img width="478" alt="image" src="https://user-images.githubusercontent.com/113132642/227634792-a9be7892-bea5-4eb9-ba17-700ae5796bac.png">

Reply:
> dumb reg ex q. Seems like they moved to js style regular expressions. So did it just change for a|b -> a or b to a||b -> a or b?
> it seems so! According to https://github.com/microsoft/vscode/issues/175540 and [this page](https://code.visualstudio.com/api/references/when-clause-contexts)

Hmm, would you mind pointing which wording made you think so? Because we didn't change how we interpret regex patterns, it's just I probably had worded myself poorly.

Also, I think you don't need to change a when clause if there are no errors underlined.